src/checkbox_todo.js

```diff
@@ -0,0 +1,10 @@
+export function toggleChkBox(todos, index) {
+  todos[index].completed = !todos[index].completed;
+  localStorage.setItem('todos', JSON.stringify(todos));
+}
+
+export const clearCompletedChkBox = (todos) => {
+  todos = todos.filter((todo) => !todo.completed);
+  localStorage.setItem('todos', JSON.stringify(todos));
+  window.location.reload();
```

Comment:
maybe its possible find another solution for not reload the page, this goes against user experience.